cvat-ui/src/components/annotation-page/review/issues-aggregator.tsx

```diff
@@ -255,7 +265,12 @@ export default function IssueAggregatorComponent(): JSX.Element | null {
                     resolved={issueResolved}
                     highlight={highlight}
                     blur={blur}
-                    onClick={() => {
+                    onClick={(event) => {
+                        const clickCoords = {
+                            x: event.clientX,
+                            y: event.clientY,
+                        };
+                        setClickCoordinates(clickCoords);
```

Comment:
```suggestion
                        setClickCoordinates({
                            x: event.clientX,
                            y: event.clientY,
                        });
```

Reply:
Removed the client click listener in favour of transformFromSVG